embassy-usb/src/driver.rs

```diff
@@ -57,18 +56,24 @@ pub trait Driver<'a> {
 }
 
 pub trait Bus {
+    type EnableFuture<'a>: Future<Output = ()> + 'a
+    where
+        Self: 'a;
     type PollFuture<'a>: Future<Output = Event> + 'a
+    where
+        Self: 'a;
+    type RemoteWakeupFuture<'a>: Future<Output = Result<(), Unsupported>> + 'a
     where
         Self: 'a;
 
-    fn poll<'a>(&'a mut self) -> Self::PollFuture<'a>;
+    /// Enables the USB peripheral. Soon after enabling the device will be reset, so
+    /// there is no need to perform a USB reset in this method.
+    fn enable(&mut self) -> Self::EnableFuture<'_>;
+
+    /// Disables and powers down the USB peripheral.
+    fn disable(&mut self);
```

Comment:
disable() should be async as well, for consistency

Reply:
sure